docs/en/multi_modal/xcomposer2d5.md

```diff
@@ -0,0 +1,160 @@
+# InternLM-XComposer-2.5
+
+## Introduction
+
+[InternLM-XComposer-2.5](https://github.com/InternLM/InternLM-XComposer) excels in various text-image comprehension and composition applications, achieving GPT-4V level capabilities with merely 7B LLM backend. IXC-2.5 is trained with 24K interleaved image-text contexts, it can seamlessly extend to 96K long contexts via RoPE extrapolation. This long-context capability allows IXC-2.5 to perform exceptionally well in tasks requiring extensive input and output contexts. LMDeploy LMDeploy supports model [internlm/internlm-xcomposer2d5-7b](https://huggingface.co/internlm/internlm-xcomposer2d5-7b)  in TurboMind engine.
+
+## Quick Start
+
+### Installation
+
+Install LMDeploy with pip (Python 3.8+). Refer to [Installation](https://lmdeploy.readthedocs.io/en/latest/get_started.html#installation) for more.
+
+```shell
+pip install lmdeploy
+
+# install other packages that InternLM-XComposer-2.5 needs
+pip install decord
+```
+
+### Offline inference pipeline
+
+The following sample code shows the basic usage of VLM pipeline. For more examples, please refer to [VLM Offline Inference Pipeline](https://lmdeploy.readthedocs.io/en/latest/inference/vl_pipeline.html#vlm-offline-inference-pipeline)
+
+```python
+from lmdeploy import pipeline
+from lmdeploy.vl import load_image
+from lmdeploy.vl.constants import IMAGE_TOKEN
+
+pipe = pipeline('internlm/internlm-xcomposer2d5-7b')
+
+image = load_image('https://raw.githubusercontent.com/open-mmlab/mmdeploy/main/tests/data/tiger.jpeg')
+response = pipe((f'describe this image {IMAGE_TOKEN}', image))
```

Comment:
Do we have to specify `{IMAGE_TOKEN}`?

Reply:
xcomposer2 doesn't restrict the location of image_token, so so I think it would be better to show this in the example.